### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by tkoyama010**
### Overview

Fix typo

 

### Details

- None

### CODE REVIEW

**Review by adeak — DISMISSED** [review]
LGTM, thanks!

File: examples/02-plot/lookup-table.py [hunk 1]
```diff
@@ -23,7 +23,7 @@
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 # First, let's plot using the default color map, "viridis". Internally, PyVista
 # will automatically create a lookup table to map the scalars (stored here
-# within `point_data`, to RGBA colors. This is shown here as a nested attribute
+# within `point_data`, to RGBA colors). This is shown here as a nested attribute
```
Comment (MatthewFlamm) [inline examples/02-plot/lookup-table.py:26]:
```suggestion
# within `point_data`) to RGBA colors. This is shown here as a nested attribute
```

**Review by adeak — APPROVED** [review]
Looks even gooder to me now, thanks :sweat_smile:

**Review by MatthewFlamm — APPROVED** [review]
LGTM

**Review by banesullivan — APPROVED** [review]

**Pull request merged**